example/basic-delivery/index.js

```diff
@@ -0,0 +1,12 @@
+const { Delivery } = require('../../delivery')
+const { Source } = require('../../source')
```

Comment:
The only thing I would really suggest is that the examples import the top level export, rather than individual files.  Want me to make that change?

Reply:
Yeah go for it! I wrote the top level `index.js` exports after the examples and never got around to it